### PR TITLE
Support string flag

### PR DESCRIPTION
**Pull request by axinging**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7509)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @xhcao @gyagp PTAL

File: tfjs-core/src/environment.ts [hunk 1]
```diff
@@ -192,9 +196,9 @@ function parseValue(flagName: string, value: string): FlagValue {
     return value === 'true';
   } else if (`${+ value}` === value) {
     return +value;
+  } else {
+    return value;
```
Comment (gyagp) [inline tfjs-core/src/environment.ts:200]: for string, I'd prefer case sensitive.

**Review by pyu10055 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7509)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @gyagp and @mattsoulanille)

___
*[`tfjs-core/src/environment.ts` line 200 at r1](https://reviewable.io/reviews/tensorflow/tfjs/7509#-NRC6b9-BbEDp64VwP8Q-r1-200:-NRZMDAP6ZD3stc_XSSi:b32v57) ([raw file](https://github.com/tensorflow/tfjs/blob/edbf6450dc6430c16469e26873aa00d6fb507bcc/tfjs-core/src/environment.ts#L200)):*
<details><summary><i>Previously, gyagp (Yang Gu) wrote…</i></summary><blockquote>

For string, I'd prefer case sensitive.
</blockquote></details>

+1

File: tfjs-core/src/environment.ts (hunk 1 above)
Comment (mattsoulanille) [reply tfjs-core/src/environment.ts:200]: I agree with Yang. Case sensitive would be better for strings.

**Review by mattsoulanille — CHANGES_REQUESTED** [review]

**Comment by axinging** [comment]
@gyagp  @mattsoulanille 
This is case sensitive.  I think the test case is misleading. So I refined the test case, PTAL

**Review by mattsoulanille — CHANGES_REQUESTED** [review]

**Comment by axinging** [comment]
@gyagp  @mattsoulanille , now URL string is also case seneitive, PTAL

**Review by gyagp — APPROVED** [review]
LGTM

**Review by mattsoulanille — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**